AI/src/main/java/dk/gruppeseks/bodtrd/ai/Node.java

```diff
@@ -42,12 +42,12 @@ public Node(int gridX, int gridY, Node parent, int depth)
         this._depth = depth;
     }
 
-    public double heuristic(int goalX, int goalY)
+    public double getTotalCost(int goalX, int goalY)
```

Comment:
imo we should have a private heuristic method returning `c`
however i dont see how a, b, and c are descriptive in any manner.
perhaps `distanceX`, `distanceY`, `directLineDistance`?